src/main/java/hello/TaskRepository.java

```diff
@@ -7,52 +7,42 @@
 
 /**
  * Created by Vladyslav Usenko on 16.01.2016.
+ * @deprecated it is now just wrapper around a map, consider removing this class completely
  */
 @Repository
+@Deprecated
 public class TaskRepository {
 
-    private final Map<UUID, Task> repository;
+    private final Map<UUID, Task> repository = new ConcurrentHashMap<UUID, Task>();
 
-    public TaskRepository() {
-        this.repository = new ConcurrentHashMap<UUID, Task>();
-    }
-
-    public synchronized void store(Task task){
+    public void store(Task task){
         repository.put(task.getId(), task);
     }
 
-    public synchronized Task load(UUID id){
-        return repository.get(id);
+    public Optional<Task> load(UUID id){
```

Comment:
Optional explicitly shows method contract, that it is expected to return null in some cases